README.md

```diff
@@ -13,22 +13,40 @@ Our implementation is based on [CSBDEEP](http://csbdeep.bioimagecomputing.com) (
 
 ## Installation
 This implementation requires [Tensorflow](https://www.tensorflow.org/install/).
-We have tested Noise2Void using Python 3.7 and tensorflow-gpu 2.4.1.
+We have tested Noise2Void using Python 3.9 and TensorFlow 2.7 and 2.10.
 
-Note: If you want to use TensorFlow 1.15 you have to install N2V v0.2.1. N2V v0.3.0 supports TensorFlow 2 only.
+Note: If you want to use TensorFlow 1.15 you have to install N2V v0.2.1. N2V v0.3.* supports TensorFlow 2 only.
 
 #### If you start from scratch...
 We recommend using [miniconda](https://docs.conda.io/en/latest/miniconda.html).
 If you do not yet have a strong opinion, just use it too!
 
-After installing Miniconda, the following lines might are likely the easiest way to get Tensorflow and CuDNN installed on your machine (_Note:_ Macs are not supported, and if you sit on a Windows machine all this might also require some modifications.):
+After installing Miniconda, the following lines might are likely the easiest way to get Tensorflow and 
+CuDNN installed on your machine.
+_Note:_ Macs are not GPU-supported with TensorFlow.
 
+The [Tensorflow guidelines](https://www.tensorflow.org/install/pip) provide a step by step guide on how to install
```

Comment:
Maybe we can point directly to the step-by-step instructions: https://www.tensorflow.org/install/pip#step-by-step_instructions

In that case I would also vote to remove any install instructions for tensorflow. I was kind of confused and started following the **For linux:** instructions :innocent:

Reply:
Yes that would make more sense